pkg/cloud/services/secretsmanager/helpers.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package secretsmanager
+
+func splitBytes(data []byte, maxSize int, iterFunc func([]byte)) {
```

Comment:
Needs to change to this https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1535#discussion_r376679961, but can be done as follow up.